server/src/main/java/org/elasticsearch/index/mapper/KeywordFieldMapper.java

```diff
@@ -922,6 +923,10 @@ private void indexValue(DocumentParserContext context, String value) {
             return;
         }
 
+        if (hasNormalizer() && fieldType().isSyntheticSource) {
```

Comment:
Can `storeIgnored` field be used here instead of `fieldType().isSyntheticSource`? I see that `storeIgnored` set to true when synthetic source is used.

Reply:
I thought about that but I didn't like using `storeIgnored` because reading the code looks as if we are storing it because the field value is ignored...which in my opinion is a bit misleading. The efefct is the same but the reason why we use one option or the other is not.

Reply:
@lkts

Reply:
I mean `storeIgnored` is just set to `context.isSourceSynthetic()`, can we just rename it?